tools/formatter/FormatterTool.java

```diff
@@ -147,21 +166,25 @@ private static boolean shouldFormatFile(Path file, Set<Path> ignoredDirectories)
         ((isJava(file) && java.get())
             || (isProto(file) && proto.get())
             || (isPython(file) && python.get())
-            || (isCpp(file) && cpp.get()));
+            || (isCpp(file) && cpp.get())
+            || (isBuild(file) && build.get()));
     boolean inIgnoredDirectory =
         ignoredDirectories.stream().anyMatch(ignoredDir -> file.startsWith(ignoredDir));
     return formatByExtension && !inIgnoredDirectory;
   }
 
   private static Map<String, BaseFormatter> formatters =
-      ImmutableMap.of(
-          ".java", JavaFormatter.init(),
-          ".proto", ClangFormatter.init(),
-          ".py", PythonFormatter.init(),
-          ".cc", ClangFormatter.init());
+      ImmutableMap.<String, BaseFormatter>builder()
+          .put(".java", JavaFormatter.init())
+          .put(".proto", ClangFormatter.init())
+          .put(".py", PythonFormatter.init())
+          .put(".cc", ClangFormatter.init())
+          .put(".bazel", BuildFormatter.init())
+          .put("", BuildFormatter.init())
```

Comment:
No extension it formatted as a BUILD file?

Reply:
@oferb 
If file does not have an extension but filename is not equal to `BUILD`, it would *not* be formatted.
It's the limitation of having extensions mapped to formatters (`ImmutableMap<String[extension], BaseFormatter>`). A refactor we could do is change it to `ImmutableMap<Pattern, BaseFormatter`) but then performance drops due to matching filenames via regex.

Reply:
How about a refactor (in a separate PR) to:
- Remove `shouldFormatFile()`
- Remove `formatters`
- Instead, have a method `getFormatter()` with the same inputs as `shouldFormatFile`
- Use this input in main loop, instead of `shouldFormatFile`

This would remove the duplication we have now where we have e.g `.java` both in `isJava()` and in `formatters`.